### ISSUE TITLE
Track captions for video tags

### ISSUE THREAD

**Issue by freducom** [flipsite/flipsite#7]
The YAML should have support for a list of <track> to add to a <video> tag for src, srclang and label

More: https://dequeuniversity.com/rules/axe/3.5/video-caption?application=AxeChrome
